### PR TITLE
doc: deliverables docs

### PR DESCRIPTION
**Pull request by Rqnsom**
- small improvements in readme, design document
- deliverable folder and all documents related to deliverables
- old testing guide (M1) deprecated

### CODE REVIEW

File: doc/final-design.md [hunk 1]
```diff
@@ -151,10 +157,19 @@ Get module binary using account address.
 
 # Design architecture
 
+The main parts are:
+- PalletMove _(this repo)_.
```
Comment (asmie) [inline doc/final-design.md:161]: Sometimes it's `pallet-move` and now here we have `PalletMove`.
Let's choose one naming and use it consequently.

File: doc/final-design.md [hunk 2]
```diff
@@ -151,10 +157,19 @@ Get module binary using account address.
...
+- The `no-std` MoveVM fork adapted for the Substrate framework.
+  - It can be found inside the _[substrate-move][substrate-move]/language/_ directory.
+- The backend layer which is an interface between the MoveVM and the pallet.
+  - It is located inside the _[substrate-move][substrate-move]/_ directory.
+- [smove][smove] tool, which is necessary for the compilation of Move source code.
```
Comment (asmie) [inline doc/final-design.md:166]: It's really necessary for **the compilation**?

File: doc/tech_guide.md [hunk 3]
```diff
@@ -65,7 +65,11 @@ Verification for Rust's linter `clippy` by executing:
 cargo clippy --all-targets -- -D warnings
 ```
 
-Read about internal test concepts (what tests cover and how) in the [testing guide](testing_guide.md).
+_Info: All above instructions are executed in our CI workflow._
```
Comment (asmie) [inline doc/tech_guide.md:68]:
```suggestion
_Info: All the above instructions are executed in our CI workflow._
```

File: README.md [hunk 4]
```diff
@@ -70,25 +78,27 @@ _There are more potential improvements to be made._
...
 - Develop full benchmarks
-- Multi-signature support
+- Multi-signature support!
 - Protection mechanism for Move address 0x1
-- Add documentation about Substrate-stdlib
-- Restructure GitHub repositories of Substrate-stdlib and Move-stdlib
-
+- Move `Move-stdlib` repository location
+- Add an extrinsic for updating `Move-stdlib` with special root account
+- Improve RPC commands (and `smove` commands)
```
Comment (asmie) [inline README.md:95]: That point is unclear to me - so what's wrong with the current RPC and smove commands, someone may ask.

**Review by asmie — APPROVED** [review]
Overall looks great! :smile:

File: README.md (hunk 4 above)
Comment (Rqnsom) [reply README.md:95]: Done

File: doc/final-design.md (hunk 1 above)
Comment (Rqnsom) [reply doc/final-design.md:161]: Done

File: doc/tech_guide.md (hunk 3 above)
Comment (Rqnsom) [reply doc/tech_guide.md]: Done

File: doc/final-design.md (hunk 2 above)
Comment (Rqnsom) [reply doc/final-design.md:166]: Agreement from Slack - let's keep it like this to avoid confusion and extra info

File: README.md [hunk 5]
```diff
@@ -9,7 +9,14 @@ A pallet for substrate based blockchains to enable the usage of smart contracts
...
 
-For the execution of Move scripts and the publication of Move modules, the Move source code has to be compiled and serialized into bytecode. For this purpose, the tool [`smove`](https://github.com/eigerco/smove) is provided. The tool also provides further helpful features for developing and working with the Move language and this pallet.
+For the execution of Move scripts and the publication of Move modules, the Move source code has to be compiled and serialized into bytecode. For this purpose, the tool [`smove`][smove] is provided. The tool also provides further helpful features for developing and working with the Move language and this pallet.
+
+### Requirements
+
+- [Substrate developer environment](https://docs.substrate.io/install/)
+- [smove][smove] is a package manager for Move language in Substrate. Follow the instructions in its repo to install it.
+- [cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html)
```
Comment (neutrinoks) [inline README.md:18]: Mhh, so Docker is only necessary if you are willing to work with Docker - but the same could be true for cargo... If you directly use the Docker image, you could avoid installing cargo - couldn't you?

File: doc/milestone-deliverables/README.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Milestone deliverables
+
+Local files are kept in the folder, while the submitted milestone links are listed here below:
+- [Milestone 1][M1]: In-Depth Exploration and Assessment of MoveVM and Substrate Integration
+- Milestone 2: MoveVM compatibility work and Subtrate Pallet development
```
Comment (neutrinoks) [inline doc/milestone-deliverables/README.md:5]: Is that link to Milestone-2-markdown file missing?

**Review by neutrinoks — APPROVED** [review]
Added some comments, but overall it looks good :+1:

File: README.md (hunk 5 above)
Comment (Rqnsom) [reply README.md:18]: Well, the `smove` installation requires `cargo` so 🤷

**Review by asmie — APPROVED** [review]

**Pull request merged**